Tests/DataPipeline/DataPipelineInteractionTests.swift

```diff
@@ -94,9 +89,6 @@ class CustomerIOImplementationTest: UnitTest {
 
         customerIO.identify(identifier: givenIdentifier)
 
-        XCTAssertEqual(profileIdentifyHookMock.beforeIdentifiedProfileChangeCallsCount, 1)
```

Comment:
Wondering that we no longer make use of hooks in this test method but the name still says `test_identify_givenPreviouslyIdentifiedCustomer_expectRunHooks_expectDeleteDeviceToken`

Reply:
See my [reply above](https://github.com/customerio/customerio-ios/pull/473#discussion_r1448462344)